solidus_rails/solidus_rails.gemspec

```diff
@@ -0,0 +1,26 @@
+# encoding: UTF-8
+require_relative '../core/lib/spree/core/version.rb'
+
+Gem::Specification.new do |s|
+  s.platform    = Gem::Platform::RUBY
+  s.name        = 'solidus_rails'
+  s.version     = Spree.solidus_version
+  s.summary     = 'Provides the rails glue for the legacy solidus frontend/backend/api'
+  s.description = s.summary
+
+  s.required_ruby_version = '>= 2.1.0'
+  s.author      = 'Solidus Team'
+  s.email       = 'contact@solidus.io'
+  s.homepage    = 'http://solidus.io'
+  s.license     = 'BSD-3-Clause'
+
+  s.files        = `git ls-files`.split("\n")
+  s.require_path = 'lib'
+
+  s.required_ruby_version     = '>= 2.2.2'
+  s.required_rubygems_version = '>= 1.8.23'
+
+  s.add_dependency 'solidus_core', s.version
+
+  s.add_dependency 'rails', '~> 5.1.0'
```

Comment:
This can be split out to just actionmailer and activerecord.

Reply:
I will once the deps are actually removed, aye. There are still lots of rails calls inside the codebase (and we still have the engine in there).